components/entity/src/main/java/org/apache/ofbiz/entity/GenericDelegator.java

```diff
@@ -1130,10 +1130,11 @@ public int removeByCondition(String entityName, EntityCondition condition) throw
             GenericHelper helper = getEntityHelper(entityName);
 
             // We check if there are eca rules for this entity
-            String entityEcaReaderName = EntityEcaUtil.getEntityEcaReaderName(this.delegatorBaseName);
-            boolean hasEntityEcaRules = UtilValidate.isNotEmpty(
-                    EntityEcaUtil.getEntityEcaCache(entityEcaReaderName).get(entityName));
-
+//            String entityEcaReaderName = EntityEcaUtil.getEntityEcaReaderName(this.delegatorBaseName);
```

Comment:
Here EntityEcaUtil is from another component entity-ext .
Not sure how to deal with this. 
But it's not ok to depend on that component since that package builds on entity .